queue/src/fiskaltrust.Middleware.Storage.AzureTableStorage/Repositories/AzureTableStorageQueueItemRepository.cs

```diff
@@ -97,15 +97,30 @@ public override async Task InsertAsync(ftQueueItem storageEntity)
             await _receiptReferenceIndexRepository.InsertAsync(new ReceiptReferenceIndex { cbReceiptReference = storageEntity.cbReceiptReference, ftQueueItemId = storageEntity.ftQueueItemId });
         }
 
-        public override async Task InsertOrUpdateAsync(ftQueueItem storageEntity)
+
+        public async Task InsertOrUpdateAsync(ftQueueItem storageEntity)
         {
-            await base.InsertOrUpdateAsync(storageEntity);
+            EntityUpdated(storageEntity);
+            var entity = MapToAzureEntity(storageEntity);
+            await _tableClient.UpsertEntityAsync(entity, TableUpdateMode.Replace);
             await _receiptReferenceIndexRepository.InsertOrUpdateAsync(new ReceiptReferenceIndex { cbReceiptReference = storageEntity.cbReceiptReference, ftQueueItemId = storageEntity.ftQueueItemId });
         }
 
+        public IAsyncEnumerable<ftQueueItem> GetByTimeStampRangeAsync(long fromInclusive, long toInclusive)
+        {
+            var result = _tableClient.QueryAsync<TableEntity>(filter: TableClient.CreateQueryFilter($"TimeStamp ge {fromInclusive} and TimeStamp le {toInclusive}"));
```

Comment:
Not sure if we are able to use Linq here, but I am afraid that the answer is potentially now. What we should still do though is use `nameof` to make sure that we don't mix the `Timestamp` from the TableEntity with the one from QueueItem because it is just different in casing